server/db.js

```diff
@@ -7,6 +7,8 @@ const localDb = ["0.0.0.0", "127.0.0.1", "localhost"].includes(
 	new URL(config.dbUrl).hostname
 );
 
+export { pool };
+
```

Comment:
This is not what I meant. I meant to add it to what was originally line `10`, so it ends up like this:

```
export const pool = new Pool({
```